client/src/pages/signin-test.tsx

```diff
@@ -37,9 +37,9 @@ const TestPostRequestPage = () => {
   const handleTestRegisterRequest = () => {
     axios
       .post("http://localhost:8000/api/app/register/", {
-        email: "user123@example.com",
+        email: "user1234567il3@example.com",
         password: "user123abc!@#",
-        username: "user123",
+        username: "useerghjr123",
```

Comment:
any reason for changing this?

Reply:
> any reason for changing this? 

I actually don't remember why or how I changed it. I probably was trying to test what it's used for or something? That whole file should be deleted actually. On the original branch I implemented functional API calls so there's no need for this, I'll remove it

Reply:
Wait no, idk what this page is used for. Probably leave it on main for now and we can delete all test pages later once the APIs calls are implemented